proto/aquila/action_execute.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+option ruby_package = "Tucana::Aquila";
+
+package aquila;
+
+message ActionCallRequest {
```

Comment:
Adapter calls Aquila with paramters

Reply:
Missing identification for action function to execute